storybookjs/react-native
# feat: check control

Issue: #541 

## What I did

Added a check control type

## How to test

run the example app and go to the check story 

- Does this need a new example in examples/expo-example? yes
- Does this need an update to the documentation? yes

If your answer is yes to any of these, please make sure to include it in your PR.
